.github/workflows/deploy.yml

```diff
@@ -111,3 +115,29 @@ jobs:
           asset_path: ./distribution/effekt.tgz
           asset_name: effekt.tgz
           asset_content_type: application/gzip
+
+  publish-npm:
+    name: Publish NPM Package
+    runs-on: ubuntu-latest
+    needs: [build-jar, release]
+    steps:
+      - name: Download npm package
+        uses: actions/download-artifact@v3
+        with:
+          name: effekt-npm-package
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '12.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Publish to NPM as effekt-lang
+        run: npm publish effekt.tgz --access public
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Publish to NPM as @effekt-lang/effekt
+        run: npm publish effekt.tgz --access public --scope @effekt-lang
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I haven't tested this; I never even published an NPM package myself, so this is just what I read in the docs.
I'm quite unsure whether this will work, but I'm fine with tweaking this later. :)

If someone with access is bored, could you try it with `0.2.2`: it's released, but _not_ on `npm` yet, which makes it a perfect candidate.

Also, we need to add the NPM token for `effekt-lang` and `@effekt-lang/effekt` to the repository secrets.